app/src/main/java/com/codesoom/assignment/dto/ErrorResponse.java

```diff
@@ -0,0 +1,13 @@
+package com.codesoom.assignment.dto;
+
+public class ErrorResponse {
+    private String message;
```

Comment:
기왕이면 `final`로 선언해 주는 것이 좋겠습니다.
```suggestion
    final private String message;
```